client/my-sites/stats/stats-list/stats-list-actions.jsx

```diff
@@ -0,0 +1,102 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import titlecase from 'to-title-case';
+import Follow from './action-follow';
+import OpenLink from './action-link';
+import Page from './action-page';
+import Promote from './action-promote';
+import Spam from './action-spam';
+
+const StatsListActions = ( { data, moduleName, translate } ) => {
+	const [ isDisabled, setIsDisabled ] = useState( false );
+	let actionList;
+	const moduleNameTitle = titlecase( moduleName );
+	const actionMenu = data.actionMenu;
+	const actionClassSet = classNames( 'stats-list-actions', 'module-content-list-item-actions', {
+		collapsed: actionMenu && ! isDisabled,
+	} );
+
+	if ( data.actions ) {
+		const actionItems = [];
+
+		data.actions.forEach( ( action ) => {
+			let actionItem;
+
+			switch ( action.type ) {
+				case 'follow':
+					if ( action.data ) {
+						actionItem = (
+							<Follow
+								key={ action.type }
+								moduleName={ moduleNameTitle }
+								isFollowing={ !! action.data.is_following }
+								siteId={ action.data.blog_id }
+								translate={ translate }
```

Comment:
Do I need to wrap the components with `localize()` even if I'm only passing the method down?

Reply:
Thanks. All of the components are wrapped in localize so I can drop the props here.